components/dashboards-web-component/src/DashboardCreatePage.jsx

```diff
@@ -0,0 +1,177 @@
+                               textareaStyle={textareaStyle} underlineStyle={underlineStyle} style={textFieldStyle}
+                               hintText="E.g. sales-stats"/>
+                </div>
+                <div class="form-group" className="create-dashboard-form-group">
+                    <label className="create-dashboard-page-label-description">
+                        Description <span class="required">*</span>
+                    </label>
+                    <TextField id="dashboardDescription" hintStyle={hintStyle} textareaStyle={textareaStyle}
+                               underlineStyle={underlineStyle} style={textFieldStyle}
+                               hintText="E.g. Monthly Sales Statistics"/>
+                </div>
+                <RaisedButton onClick={this.addDashboard} label="Add" primary={true}
+                              className="create-dashboard-button"/>
+                <Snackbar contentStyle={messageBoxStyle} bodyStyle={this.state.messageStyle} open={this.state.showMsg}
+                          message={this.state.Message} autoHideDuration={4000}
+                          onRequestClose={this.handleRequestClose}/>
+            </div>
+        </MuiThemeProvider>;
+    }
+
+    handleURL(event, value) {
+        this.setState({
+            url: this.sanitizeInput(value.toLowerCase().replace(/\s+/g, ''))
+        });
+    }
+
+    sanitizeInput(input) {
+        return input.replace(/[^a-z0-9-\s]/gim, "");
+    };
+
+    handleRequestClose() {
+        this.setState({
+            showMsg: false,
+            Message: ""
+        });
+    }
+
+    showError(message) {
+        this.setState({
+            messageStyle: errorMessageStyle,
+            showMsg: true,
+            Message: message
+        });
+    }
+
+    showMessage(message) {
+        this.setState({
+            messageStyle: successMessageStyle,
+            showMsg: true,
+            Message: message
+        });
+    }
+
+    addDashboard() {
+        let dashboardName = document.getElementById("dashboardName").value;
+        let dashboardURL = document.getElementById("dashboardURL").value;
+        let dashboardDescription = document.getElementById("dashboardDescription").value;
+        let dashboardJson = {};
+        dashboardJson.url = dashboardURL;
+        dashboardJson.name = dashboardName;
+        dashboardJson.description = dashboardDescription;
+        dashboardJson = this.generateDashboardJSON(dashboardJson);
+        let dashboardAPIs = new DashboardAPIS();
+        var that = this;
+        if (dashboardURL && dashboardName) {
+            dashboardAPIs.getDashboardByID(dashboardURL).then(function (response) {
+                if (typeof response.data === "string") {
+                    dashboardAPIs.createDashboard(dashboardJson).then(function (response) {
+                        if (response.status === 201) {
+                            that.showMessage("Dashboard is created successfully !!");
```

Comment:
Let's say Dashboard X created successfully (mention the name of the dashboard created)